lib/tasks/scripts/purge_users_data.rb

```diff
@@ -0,0 +1,16 @@
+module PurgeUsersData
+  def self.perform
+    return if ENV['SIMPLE_SERVER_ENV'] == 'production'
+
+    models = [BloodPressure, BloodSugar, Appointment, CallLog, Communication, PrescriptionDrug,
+              Observation, Encounter, ExotelPhoneNumberDetail, TwilioSmsDeliveryDetail, MedicalHistory,
+              PatientBusinessIdentifier, PatientPhoneNumber, Patient, Address]
+
+    ActiveRecord::Base.transaction do
+      models.each do |model|
+        puts "Deleting #{model} data"
+        model.delete_all
```

Comment:
Note that this won't invoke `destroy` based callbacks. Not sure how much we are using them.  

Do we have dependent destruction setup, either via AR or at the DB level?

Reply:
We only have [1 dependent destroy](https://github.com/simpledotorg/simple-server/blob/master/app/models/organization.rb#L4) in the system which is a resource that is intentionally not a part of this purge.

We've mostly never needed dependent destroys (or destroy callbacks) throughout because very few important things are actually deletable by users.

Reply:
Cool. Totally not necessary right now, but long term we should probably start to add dependent destroys in a sensible way throughout the code base. Given we are dealing with sensitive data, we are going to have to handle safely and cleanly deleting patient data at some point I'm sure.